src/ui/buttons/props.js

```diff
@@ -430,7 +430,7 @@ export type ApplePaySessionConfigRequest = (
 
 export type ButtonMessage = {|
   amount?: number,
-  offer?: $ReadOnlyArray<$Values<typeof MESSAGE_OFFER>>,
+  offer?: $Values<typeof MESSAGE_OFFER> | string,
```

Comment:
```suggestion
  offer?: $Values<typeof MESSAGE_OFFER>
```

The string type is implicit by the enum values of `MESSAGE_OFFER`